src/lib/storage/value_segment.cpp

```diff
@@ -14,32 +14,27 @@ namespace opossum {
 
 template <typename T>
 AllTypeVariant ValueSegment<T>::operator[](const ChunkOffset chunk_offset) const {
-  // Implementation goes here
-  Fail("Implementation is missing.");
+  return _stored_values.at(chunk_offset);
```

Comment:
The overloaded operator is not supposed to throw exceptions. With `array.at()` you can do so.